### PR TITLE
New PR with changes to Readout and Request models reverted, timestamps switched to nanoseconds

### PR DESCRIPTION
**Pull request by willpvazquez**

### CODE REVIEW

File: include/readout/NDReadoutTypes.hpp [hunk 1]
```diff
@@ -60,11 +54,42 @@ struct PACMAN_MESSAGE_STRUCT
...
+
+      std::cout << "Inspecting packet" << std::endl;
+
+      std::cout << "Packet Type: " << thePacket->data_packet.type << std::endl;
+      std::cout << "Packet Chip ID: " << thePacket->data_packet.chipid << std::endl;  
+      std::cout << "Packet Channel ID: " << thePacket->data_packet.channelid << std::endl;
+
+      std::cout << "packet timestamp: " << thePacket->data_packet.timestamp<< std::endl;
+    }
+  }
```
Comment (floriangroetschla) [inline include/readout/NDReadoutTypes.hpp:92]: Try to use `TLOG` instead of `std::cout`

**Review by floriangroetschla — COMMENTED** [review]
Looks good to me, some minor linter and code style issues can still be fixed later but should be addressed at some point

File: include/readout/NDReadoutTypes.hpp (hunk 1 above)
Comment (willpvazquez) [reply include/readout/NDReadoutTypes.hpp]: Have replaced with TLOG in latest commit. I also addressed some of the lint warnings. Removing c-style casts may take more significant modifications to the underlying frame, so will review this during a later iteration.

**Review by roland-sipos — APPROVED** [review]
LGTM

**Comment by roland-sipos** [comment]
Thanks for the fix, merging.

**Pull request merged**